Index: node_modules/.pnpm/@sphereon+ssi-types@0.21.2-next.8/node_modules/@sphereon/ssi-types/dist/mapper/credential-mapper.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/node_modules/.pnpm/@sphereon+ssi-types@0.21.2-next.8/node_modules/@sphereon/ssi-types/dist/mapper/credential-mapper.js b/node_modules/.pnpm/@sphereon+ssi-types@0.21.2-next.8/node_modules/@sphereon/ssi-types/dist/mapper/credential-mapper.js
--- a/node_modules/.pnpm/@sphereon+ssi-types@0.21.2-next.8/node_modules/@sphereon/ssi-types/dist/mapper/credential-mapper.js	
+++ b/node_modules/.pnpm/@sphereon+ssi-types@0.21.2-next.8/node_modules/@sphereon/ssi-types/dist/mapper/credential-mapper.js	(date 1712867694488)
@@ -311,7 +311,7 @@
         return '@context' in presentation && (((_a = presentation.type) === null || _a === void 0 ? void 0 : _a.includes('VerifiablePresentation')) || false);
     }
     static isSdJwtDecodedCredentialPayload(credential) {
-        return 'vct' in credential;
+        return typeof credential !== 'string' && 'vct' in credential;
     }
     static areOriginalVerifiableCredentialsEqual(firstOriginal, secondOriginal) {
         // String (e.g. encoded jwt or SD-JWT)
@@ -618,4 +618,4 @@
 CredentialMapper.isWrappedSdJwtVerifiablePresentation = types_1.isWrappedSdJwtVerifiablePresentation;
 CredentialMapper.isWrappedW3CVerifiableCredential = types_1.isWrappedW3CVerifiableCredential;
 CredentialMapper.isWrappedW3CVerifiablePresentation = types_1.isWrappedW3CVerifiablePresentation;
-//# sourceMappingURL=credential-mapper.js.map
\ No newline at end of file
+//# sourceMappingURL=credential-mapper.js.map
