Index: node_modules/.pnpm/@sphereon+oid4vci-issuer@0.10.1_msrcrypto@1.5.8/node_modules/@sphereon/oid4vci-issuer/dist/VcIssuer.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/node_modules/.pnpm/@sphereon+oid4vci-issuer@0.10.1_msrcrypto@1.5.8/node_modules/@sphereon/oid4vci-issuer/dist/VcIssuer.js b/node_modules/.pnpm/@sphereon+oid4vci-issuer@0.10.1_msrcrypto@1.5.8/node_modules/@sphereon/oid4vci-issuer/dist/VcIssuer.js
--- a/node_modules/.pnpm/@sphereon+oid4vci-issuer@0.10.1_msrcrypto@1.5.8/node_modules/@sphereon/oid4vci-issuer/dist/VcIssuer.js	
+++ b/node_modules/.pnpm/@sphereon+oid4vci-issuer@0.10.1_msrcrypto@1.5.8/node_modules/@sphereon/oid4vci-issuer/dist/VcIssuer.js	(date 1712868209137)
@@ -224,7 +224,7 @@
                         };
                     }
                 }
-                if (did && !ssi_types_1.CredentialMapper.isSdJwtDecodedCredentialPayload(credential) && !!ssi_types_1.CredentialMapper.isJwtEncoded(credential)) {
+                if (did && !ssi_types_1.CredentialMapper.isSdJwtDecodedCredentialPayload(credential) && !ssi_types_1.CredentialMapper.isJwtEncoded(credential)) {
                     const credentialSubjects = Array.isArray(credential.credentialSubject) ? credential.credentialSubject : [credential.credentialSubject];
                     credentialSubjects.map((subject) => {
                         if (!subject.id) {
